bin/test

```diff
@@ -4,6 +4,10 @@ require "pathname"
 require "optparse"
 require 'yaml'
 
+excluded_gems = %w[
+  sidekiq-pro
+]
+
```

Comment:
`bin/test` automatically skips running tests if `_test/` directory is not found. So you don't need to manage this list. How about removing the directory?
https://github.com/ruby/gem_rbs_collection/blob/569dd4879c76d8ed869d799a3d2ec916727ffbb2/bin/test#L181-L186